### ISSUE TITLE
Letter head missing in emailed documents since 6.27 

### ISSUE THREAD

**Issue by pcharlc** [frappe/erpnext#5199]
After updating to 6.27, whenever we email with a document print attached:
- the attached PDF contains no letterhead
- the HTML version that is viewable via the link enclosed in the email contains no letter head either.

**Comment by pcharlc** [comment]
This seems to happen irrespective of whether a letterhead is assigned to the particular company or whether it should be using default letterhead

**Comment by rmehta** [comment]
Share your letterhead HTML

**Comment by pcharlc** [comment]
It can be anything, even just:

Letterhead

**Comment by pcharlc** [comment]
Also just to clarify further:  when viewing the document within the interface (via the 'printer' icon -- e.g. during editing, after submission etc), the letterhead is present.  If rendered to PDF (by following PDF link from within view screen), the letterhead is present.   However, the letterhead is just completely absent in the *emailed* version of the document (both in the PDF attached to the email as well as in the web view of the document via the "view this in your browser" link.

Tested in 6.27.5 and 6.27.7, same problem in both -- and this is irrespective of what HTML is in the letterhead, down to that HTML being a single word like "letterhead"

**Comment by vjFaLk** [comment]
Hi, 

I'm unable to reproduce this. These are the steps I took : 

1. Created a new Letterhead and set it as default
2. Emailed Lead information to myself

Letterhead is visible in PDF and "View in browser"

**Comment by pcharlc** [comment]
Hi,

Very strange: exact same sequence as you've followed reproduces the problem for me with leads.  Also seeing this with every other doctype I've tried (including sales invoices and quotations).

This behaviour started when we upgraded from 6.26 to 6.27 as it was working before.  Wondering if something went wrong in the upgrade.  It is a VM from your VM images (Dec last year) that has been periodically bench updated; I could create a clone of it and provide you remote access to take a look directly -- what is the best way to get credentials to you?

**Comment by vjFaLk** [comment]
Hey @pcharlc we just pushed v6.27.8, can you update and check if it's working properly now?

**Comment by pcharlc** [comment]
@vjFaLk I've done a bench update and emailed myself a few docs, but it looks like email transmission is broken completely now...  Seeing errors in worker.error.log indicating that an "enqueue_events()" takes 2 args but only 1 given.   I attach the traceback via screenshot here, all the subsequent tracebacks are the same.

Also, I've noticed an xmlrpc error during every bench update since going from 6.26 to 6.27, and do not recall seeing that before, have attached a screenshot of that also.
![screen shot 2016-04-14 at 10 03 16 am](https://cloud.githubusercontent.com/assets/17624425/14536915/99cd6692-0229-11e6-9a78-cb81fdb2a9d5.png)
![screen shot 2016-04-14 at 10 08 51 am](https://cloud.githubusercontent.com/assets/17624425/14536916/99e7a5ca-0229-11e6-9c0e-808375d56086.png)

**Comment by pcharlc** [comment]
Hmm, another issue I am now having (after today's bench update from 6.27.7 to .8) is that if I view any customer, the information is not loading  (name, billing currency etc) -- I just have "Loading..." in the header bar of the form

**Comment by anandpdoshi** [comment]
@pcharlc are you using frappe's `develop` branch?

**Comment by pcharlc** [comment]
No, not that I'm aware of!  I downloaded the ERPNext-221215 vm image last year, and have only been doing "bench update" in it every few weeks after new releases have been out for a few days.

**Comment by pcharlc** [comment]
Ok, so I just looked inside the various folders inside that 221215 image, and it shows apps/erpnext as branch master, but apps/frappe as branch develop

**Comment by pcharlc** [comment]
After this update to .8 things have been so broken, I have had to go back to an earlier snapshot that I took of the VM (back at 6.27.7).  So I  made a few more experiments from there:

- bench switch-to-master:   appears to work at first. However, any subsequent "bench update" after that breaks things, web app does not come back up again.

- bench switch-to-develop  (after making bench-repo to the develop branch) ; followed by bench update -- patch ; bench update (another update is required to get all workers started):  This seems to provide a working system again with a different look (the precursor to v7 right?  Some nice features in there!)  Except, no email transmission seems to happen.   If I do things in a slightly different order instead (first do bench switch-to-develop, then make bench-repo to develop branch, then bench update):  then I do get email transmission -- but still no letterhead!   Also, bench update then does not seem to be able to do a restart (sudo supervisorctl restart frappe says "no process").

So now it looks like I have a bigger problem than just the letterhead:  how do I get back to a stable production system?  If I roll back to any VM snapshot before my bench update to 6.27.7 I will lose transactions
